modules/local/virusbreakend/main.nf

```diff
@@ -5,9 +5,7 @@ process VIRUSBREAKEND {
     label 'process_high'
 
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://pub-29f2e5b2b7384811bdbbcba44f8b5083.r2.dev/singularity/gridss:2.13.2--1':
-        'nf-core/gridss:2.13.2--1' }"
+    container "quay.io/nf-core/gridss:2.13.2--1"
```

Comment:
```suggestion
    container "nf-core/gridss:2.13.2--1"
```

No need to specify `quay.io` here, as we have this in the nextflow.config: https://github.com/nf-core/oncoanalyser/blob/47d551b1afdba26055df57c88bb3e244c9e79a41/nextflow.config#L210-L216

Reply:
Ha, I did check to see if there was some built-in mechanism in Nextflow that allowed unqualified Docker URIs to resolve to quay.io before I messaged you on Slack about it but didn't think to look through the config, thanks for investigating further!

I'll go ahead and revert the relevant change.

Reply:
I missed that this was a co-commit sorry, still getting used to that feature...

Reply:
No worries